packages/core/admin/admin/src/content-manager/components/BlocksInput/BlocksContent.tsx

```diff
@@ -218,6 +240,9 @@ const baseRenderElement = ({ props, blocks, editor }: BaseRenderElementProps) =>
   const block = blockMatch || blocks.paragraph;
   const nodePath = ReactEditor.findPath(editor, props.element);
 
+  // Link is inline block
```

Comment:
just to be extra explicit:

```suggestion
  // Link is inline block so it cannot be dragged
```